shm/coordinates.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+
+class Coordinates {
+private:
+    int positionX_;
+    int positionY_;
+
+public:
+    Coordinates()
+        : positionX_(0)
+        , positionY_(0)
+    {}
+    Coordinates(int positionX, int positionY)
+        : positionX_(positionX)
+        , positionY_(positionY)
+    {}
```

Comment:
The definition should be in .cpp files. Only simple getter/setters are allowed in .hpp